Model/Client/Request/PaymentIntents/Create.php

```diff
@@ -82,6 +68,34 @@ protected function parseResponse(ResponseInterface $request): array
         ];
     }
 
+    /**
+     * @param Quote $quote
+     *
+     * @return array|null
+     */
+    private function getShippingAddress(Quote $quote): ?array
+    {
+        $shippingAddress = $quote->getShippingAddress();
+
+        if ($quote->getIsVirtual()) {
+            return null;
+        }
+
+        return [
```

Comment:
Are all empty values aligned to the Airwallex API spec?

Reply:
These are according to the Api spec as described in https://www.airwallex.com/docs/api#/Payment_Acceptance/Payment_Intents/Intro